gcc warns for invalid conversion

```shell
  cuda/bindings/_lib/ccudart/ccudart.cpp: In function 'cudaError_t __pyx_f_4cuda_8bindings_4_lib_7ccudart_7ccudart__cudaGraphExecGetFlags(cudaGraphExec_t, long long unsigned int*)':
  cuda/bindings/_lib/ccudart/ccudart.cpp:52386:108: warning: invalid conversion from 'long long unsigned int*' to 'cuuint64_t*' {aka 'long unsigned int*'} [-fpermissive]
  52386 |   __pyx_t_3 = __pyx_f_4cuda_8bindings_5_cuda_5ccuda__cuGraphExecGetFlags(((CUgraphExec)__pyx_v_graphExec), __pyx_v_flags); if (unlikely(__pyx_t_3 == ((CUresult)CUDA_ERROR_NOT_FOUND) && __Pyx_ErrOccurredWithGIL())) __PYX_ERR(0, 3814, __pyx_L1_error)
        |                                                                                                            ^~~~~~~~~~~~~
        |                                                                                                            |
        |                                                                                                            long long unsigned int*
```
(Sorry, the module path is a bit different as I notice this when working on #91, but hopefully it's easy to map back to the old path :P)

--- a/cuda_bindings/cuda/bindings/_lib/cyruntime/cyruntime.pyx.in
+++ b/cuda_bindings/cuda/bindings/_lib/cyruntime/cyruntime.pyx.in
@@ -4639,7 +4639,7 @@ cdef cudaError_t _cudaGraphExecGetFlags(cudaGraphExec_t graphExec, unsigned long
     err = m_global.lazyInitContextState()
     if err != cudaSuccess:
         return err
-    err = <cudaError_t>cydriver._cuGraphExecGetFlags(<cydriver.CUgraphExec>graphExec, flags)
+    err = <cudaError_t>cydriver._cuGraphExecGetFlags(<cydriver.CUgraphExec>graphExec, <cydriver.cuuint64_t *>flags)
     if err != cudaSuccess:
         _setLastError(err)
     return err

